net/calendar-contacts-server/Portfile

```diff
@@ -0,0 +1,527 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               github 1.0
+
+github.setup            apple ccs-calendarserver fbd0e11675cc0f64a425581b5c8398cc1e09cb6a
+git.branch              master
+
+name                    calendar-contacts-server
+# version from https://github.com/apple/ccs-calendarserver/blob/master/setup.py
+version                 9.3
+revision                0
+categories              net mail
+platforms               darwin
+supported_archs         noarch
+maintainers             {ieee.org:s.t.smith @essandess} openmaintainer
+license                 Apache
+
+description             Apple Calendar and Contacts Server configuration
+long_description ${description}. \
+    ccs-calendarserver is a standards-compliant server implementing \
+    the CalDAV and CardDAV protocols, including iMIP support. It \
+    provides a shared location on the network allowing multiple users \
+    to store and edit calendaring and contact information. This port \
+    provides a basic, working, easily modifiable configuration, \
+    previously used in macOS Server.app, and an nginx reverse proxy \
+    to handle modern crypto and isolate the backend server.
+
+fetch.type              git
```

Comment:
Since you are pulling from a specific tag, this line should be removed so the tag will actually be used.

Reply:
Done. Thanks.

Also, moving `easy_install` into `post-activate` appears to have done the trick. Just working on niggling buildbot permissions issues now…